Source/Lib/Codec/EbResourceCoordinationProcess.c

```diff
@@ -561,38 +561,34 @@ void* ResourceCoordinationKernel(void *inputPtr)
-
-                
-                contextPtr->sequenceControlSetInstanceArray[instanceIndex]->sequenceControlSetPtr->padRight = contextPtr->sequenceControlSetInstanceArray[instanceIndex]->sequenceControlSetPtr->maxInputPadRight;
-                contextPtr->sequenceControlSetInstanceArray[instanceIndex]->sequenceControlSetPtr->croppingRightOffset = contextPtr->sequenceControlSetInstanceArray[instanceIndex]->sequenceControlSetPtr->padRight;
-                contextPtr->sequenceControlSetInstanceArray[instanceIndex]->sequenceControlSetPtr->padBottom = contextPtr->sequenceControlSetInstanceArray[instanceIndex]->sequenceControlSetPtr->maxInputPadBottom;
-                contextPtr->sequenceControlSetInstanceArray[instanceIndex]->sequenceControlSetPtr->croppingBottomOffset = contextPtr->sequenceControlSetInstanceArray[instanceIndex]->sequenceControlSetPtr->padBottom;
-
-                if(contextPtr->sequenceControlSetInstanceArray[instanceIndex]->sequenceControlSetPtr->padRight != 0 || contextPtr->sequenceControlSetInstanceArray[instanceIndex]->sequenceControlSetPtr->padBottom != 0) {
-                    contextPtr->sequenceControlSetInstanceArray[instanceIndex]->sequenceControlSetPtr->conformanceWindowFlag = 1;
-                }
-                else {
-                    contextPtr->sequenceControlSetInstanceArray[instanceIndex]->sequenceControlSetPtr->conformanceWindowFlag = 0;
-                }
-                
-				inputSize = contextPtr->sequenceControlSetInstanceArray[instanceIndex]->sequenceControlSetPtr->lumaWidth * contextPtr->sequenceControlSetInstanceArray[instanceIndex]->sequenceControlSetPtr->lumaHeight;
+            if (contextPtr->sequenceControlSetInstanceArray[instanceIndex]->encodeContextPtr->initialPicture) {
+                sequenceControlSetPtr->lumaWidth    = sequenceControlSetPtr->maxInputLumaWidth;
+                sequenceControlSetPtr->lumaHeight   = sequenceControlSetPtr->maxInputLumaHeight;
+                sequenceControlSetPtr->chromaWidth  = sequenceControlSetPtr->maxInputLumaWidth >> subWidthCMinus1;
+                sequenceControlSetPtr->chromaHeight = sequenceControlSetPtr->maxInputLumaHeight >> subHeightCMinus1;
+
+                sequenceControlSetPtr->padRight             = sequenceControlSetPtr->maxInputPadRight;
+                sequenceControlSetPtr->croppingRightOffset  = sequenceControlSetPtr->padRight;
+                sequenceControlSetPtr->padBottom            = sequenceControlSetPtr->maxInputPadBottom;
+                sequenceControlSetPtr->croppingBottomOffset = sequenceControlSetPtr->padBottom;
+
+                if (sequenceControlSetPtr->padRight != 0 || sequenceControlSetPtr->padBottom != 0)
+                    sequenceControlSetPtr->conformanceWindowFlag = 1;
+                else
+                    sequenceControlSetPtr->conformanceWindowFlag = 0;
+
+                sequenceControlSetPtr->encoderBitDepth = sequenceControlSetPtr->staticConfig.encoderBitDepth;
```

Comment:
Hi Falei,

Please double check and do more verification here, because originally encoderBitDepth was only assigned to EB_8BIT and were used in CodeProfileTier() function. Thanks!

Reply:
Thanks for pointing out this. The member assignment of `encoderBitDepth` here was removed to keep consistent with the original code.